doc/rst/language/spec/records.rst

```diff
@@ -519,8 +519,13 @@ following signatures for a record ``R``:
    proc ==(lhs:R, rhs:R) : bool where lhs.type == rhs.type;
    proc !=(lhs:R, rhs:R) : bool where lhs.type == rhs.type;
 
-Each of these compare the fields, one at a time, returning ``false`` if
-the property is not satisfied by the given pair of fields.
+Other comparison operator overloads, namely ``<``, ``<=``, ``>``, and ``>=`` have
```

Comment:
I would separate this a little more from the rest of the sentence, since your separator is currently a comma and commas are used in the list itself.  I'd probably use parentheses to do so.